src/documents/invoice/invoiceV4.ts

```diff
@@ -90,6 +92,14 @@ export class InvoiceV4 extends Document {
       prediction: apiPrediction.document_type,
       valueKey: "value",
     });
+    this.referenceNumbers = apiPrediction.reference_numbers.map(
+      function (prediction: { [index: string]: any }) {
```

Comment:
You should use the `StringDict` type here rather than redefine it.

Reply:
Got it.
Do you want me to change it everywhere in this object ?

Reply:
Yes, might as well